spring-cloud-dataflow-admin-local/pom.xml

```diff
@@ -27,6 +27,14 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.hsqldb</groupId>
+			<artifactId>hsqldb</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+		</dependency>
```

Comment:
* Will remove postgresql.  Users can add their database of choice at build time. It was a nice to have.
* Hsqldb is required since we need at least one database enumerated.